lib/utils/ModuleBase.js

```diff
@@ -25,13 +26,14 @@ export default class ModuleBase {
     }
     const { moduleName } = config;
     this._app = app;
+    this._serviceUrl = serviceUrl;
     this.namespace = config.namespace;
 
     // check if native module exists as all native
-    initialiseNativeModule(this, config);
+    initialiseNativeModule(this, config, serviceUrl);
     initialiseLogger(
       this,
-      `${app.name}:${moduleName.replace('RNFirebase', '')}`
+      `${this.app.name}:${moduleName.replace('RNFirebase', '')}`
```

Comment:
Should be `this._app` or `app`

Reply:
Hmm, I will change it to app, but this.app also works because of the getter below.

Reply:
Fixed.